src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +27,13 @@ public interface MachineService {
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<Object> machines, Machine value);
```

Comment:
The `fill` method should not use raw types. Consider parameterizing the `machines` list to be `List<T>` instead of `List<Object>`, ensuring type safety.